src/components/Progress/ProgressBar.js

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './style.scss';
```

Comment:
missing space

Reply:
we didn't decide to use this convention in the ds